solr/core/src/java/org/apache/solr/update/processor/DistributedUpdateProcessor.java

```diff
@@ -1123,6 +1123,9 @@ protected boolean versionAdd(AddUpdateCommand cmd) throws IOException {
                 // we're ok if versions match, or if both are negative (all missing docs are equal), or if cmd
                 // specified it must exist (versionOnUpdate==1) and it does.
               } else {
+                if(cmd.getReq().getParams().getBool(CommonParams.FAIL_ON_VERSION_CONFLICTS, true) == false) {
+                  return true;
+                }
                 if (cmd.getReq().getParams().getBool("fs.ignoreconflicts", false)) {
                   return true;
                 }
```

Comment:
You can delete the `fs.ignoreConflicts`-- we should update the `mn` repo to send both params in the (2?) spots we use this feature.

Reply:
Yeah, I wanted to do so after ensuring that we make relevant changes in the `mn` repo. please also ensure that we pass the `_version_=-1` param

Reply:
I always just copied the jira title or git commit message, or the exact lines from the CHANGES.txt that you would have submitted with your git commit. e.g. 

`SOLR-13320: add a param failOnVersionConflicts=false to updates not fail when there is a version conflict (Noble Paul, Scott Blum)`

(Actually, it looks like you might have not updated Solr's CHANGES.txt in Apache)